### DISCUSSION TITLE
PP-OCRv3-rec移动端部署

### DISCUSSION THREAD

**Discussion by xireli** [PaddlePaddle/PaddleOCR#12855] [Q&A]
使用官方提供的配置文件训了一个PP-OCRv3-rec模型，python infer都很正常。但是使用paddleLite转换nb格式部署移动端性能大幅度减低。（PP-OCRv2-rec训练的模型转换后的效果好一些）
config: configs/rec/PP-OCRv3/ch_PP-OCRv3_rec.yml
paddleLite==2.10
转换命令：paddle_lite_opt --model_file ../paddle/PaddleOCR-release-2.5/inference/rec_v2_lite_v5.2_infer/inference.pdmodel --param_file ../paddle/PaddleOCR-release-2.5/inference/rec_v2_lite_v5.2_infer/inference.pdiparams --optimize_out ../paddle/PaddleOCR-release-2.5/inference/rec_v2_v3.4_infer/rec_v2_opt --valid_targets=arm  --optimize_out_type=naive_buffer
请问，PP-OCRv3-rec训练的模型因为algorithm: SVTR_LCNet 暂时不能移动端部署还是我做错了什么步骤？

**Comment by inorixu** [comment]
您这还是性能降低，我的直接Segmentation fault不转换了，linux下build的opt和windows下的paddle_lite_opt，python脚本都是
有什么替代方案吗？

**Comment by renshujiajia** [comment]
> 您这还是性能降低，我的直接Segmentation fault不转换了，linux下build的opt和windows下的paddle_lite_opt，python脚本都是 有什么替代方案吗？

遇到相同问题，请问你那边解决了吗
